kyuubi-common/src/test/scala/org/apache/kyuubi/MarkdownUtils.scala

```diff
@@ -50,7 +50,7 @@ object MarkdownUtils {
         withClue(s"$markdown out of date, as line ${index + 1} is not expected." +
           " Please update doc with KYUUBI_UPDATE=1 build/mvn clean test" +
           s" -pl $module -am -Pflink-provided,spark-provided,hive-provided" +
-          s" -DwildcardSuites=$agent") {
+          s" -DwildcardSuites=$agent -DskipTests") {
```

Comment:
Please try this way
```suggestion
          s" -Dtest=none -DwildcardSuites=$agent") {
```

Reply:
Well, get this point.

According to https://spark.apache.org/developer-tools.html#individual-tests, recommended use `-Dtest=none` to skip java test.

Thanks for suggestion, i'll push a commit fix.